src/clr-addons/numericfield/numeric-field.ts

```diff
@@ -120,11 +122,14 @@ export class ClrNumericField implements OnInit, OnDestroy, AfterViewChecked {
           if (event.keyCode === BACK_KEYCODE && value.substr(cursorStart - 1, 1) === this.groupingSeparator) {
             event.target.value = value.substring(0, cursorStart - 2) + value.substring(cursorStart - 1, value.length);
             event.target.selectionStart = event.target.selectionEnd = cursorStart - 1;
+            return false;
```

Comment:
is this really needed? i want to prevent aborting the normal event flow where possible.

Reply:
needed for an issue, that when you are on the left to a grouping-seperator and press DEL, 2 digits are removed